src/xrCore/xrDebug.h

```diff
@@ -78,13 +74,27 @@ class XRCORE_API xrDebug
         const char* arg1 = nullptr, const char* arg2 = nullptr);
     static void DoExit(const std::string& message);
 
+	///
+	/// Note: DbgHelp is singlethreaded, so you must synchronize calls to these functions
+	///
+	static void LogStackTrace(const char* header);
+	static xr_vector<xr_string> BuildStackTrace(u16 maxFramesCount = MaxFramesCountDefault);
```

Comment:
Apparently, `MaxFramesCountDefault` exists only to allow you to not write `512` here, which is not the case when you want to define constants. When you use `BuildStackTrace` anywhere else, you want to know its signature and default arguments, but instead of `512` you'll see `MaxFramesCountDefault` requires you open the header and see the actual value.